Document recursive chown feature (#19391)
kubernetes/website: content/en/docs/concepts/policy/pod-security-policy.md

**FSGroup** - Controls the supplemental group applied to some volumes.

**FSGroupChangePolicy** -  FSGroupChangePolicy defines behavior of changing ownership and permission of the volume

> before being exposed inside Pod. This field will only apply to volume types which support fsGroup based ownership(and permissions).

Reviewer: ```suggestion
before being exposed inside a Pod. This field only applies to volume types that support
`fsGroup` controlled ownership (and permissions).
```

> It will have no effect on ephemeral volume types such as: secret, configmaps and emptydir.If not specified - default behaviour is to always recursively change permission and ownership. 

Reviewer: Based on the website style guide, I suggest:
```suggestion
This field has no effect on ephemeral volume types such as 
[`secret`](https://kubernetes.io/docs/concepts/storage/volumes/#secret),
[`configMap`](https://kubernetes.io/docs/concepts/storage/volumes/#configmap),
and [`emptydir`](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
If you don't specify `FSGroupChangePolicy` for a durable (non-ephemeral) volume,
the kubelet always recursively changes permission and ownership.
```

> of volume. This is an alpha feature in 1.18 and feature `ConfigurableFSGroupPolicy` must be enabled to use it.

Reviewer: > This is an alpha feature in 1.18 and feature `ConfigurableFSGroupPolicy` must be enabled to use it.

I recommend:
```markdown
This is an alpha feature. To use it, enable the
[feature gate](/docs/reference/command-line-tools-reference/feature-gates]
`ConfigurableFSGroupPolicy` in the <name-of-component(s)>.
```

**FSGroup** - Controls the supplemental group applied to some volumes.

**FSGroupChangePolicy** -  FSGroupChangePolicy defines behavior of changing ownership and permission of the volume
before being exposed inside a Pod. This field only applies to volume types that support
`fsGroup` controlled ownership (and permissions).
This field has no effect on ephemeral volume types such as 
[`secret`](https://kubernetes.io/docs/concepts/storage/volumes/#secret),
[`configMap`](https://kubernetes.io/docs/concepts/storage/volumes/#configmap),
and [`emptydir`](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
If you don't specify `FSGroupChangePolicy` for a durable (non-ephemeral) volume,
the kubelet always recursively changes permission and ownership.
of volume. This is an alpha feature. To use it, enable the
[feature gate](/docs/reference/command-line-tools-reference/feature-gates]
`ConfigurableFSGroupPolicy` in the kubelet, api-server and controller-manager.

> 

Author: this is wrong place to document this. let me move this to correct place
